src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -55,6 +55,14 @@ let plugin_flag = Command.Param.return []
 
 [%%endif]
 
+let get_tracked_keypair ~logger ~read_from_env_exn ~which conf_dir pk =
+  let%bind wallets =
+    Secrets.Wallets.load ~logger ~disk_location:(conf_dir ^/ "wallets")
+  in
+  let sk_file = Secrets.Wallets.get_path wallets pk in
+  let%map kp = read_from_env_exn ~logger ~which sk_file in
```

Comment:
BP PK may not be imported on node and no keys in .mina-config/wallets/store

Reply:
Yes, there will be a failure on startup in that case:
```
FATAL ERROR

  ☠  Mina Daemon encountered a configuration error loading uptime submitter keypair.
```

Reply:
Assume that check is only if --uptime-submitter is provided? Nodes not registered for delegation shouldn't require this? Also suggest adding new requirement to release notes; I don't import wallet on my mainnet BP nodes.

Reply:
Right, that check only happens when a submitter is given.

There should be documentation provided when this feature is included.